scripts/lint.sh

```diff
@@ -85,4 +85,14 @@ for probe in $(grep -r --include=*.[ch]pp --exclude=probes.hpp --exclude=provide
     fi
 done
 
+# Check for Windows-style line endings
+output=$(find src -type f -exec dos2unix -ic {} +)
```

Comment:
How do Windows-style line endings get into our code?

Reply:
@j-tr?

Reply:
Probably some default in the editor or alike

Reply:
guilty